src/_locales/de/messages.json

```diff
@@ -0,0 +1,21 @@
+{
+    "extensionDescription": {
+        "message": "Firefox Relay makes it easy to create email aliases that forward to your real inbox. Use it to protect your online accounts from hackers and unwanted messages.",
```

Comment:
This isn't German looking.

Reply:
This is temp file that will be deleted before merging this PR. (We did [the same thing](https://github.com/mozilla/fx-private-relay/pull/815/commits/55ba4d12221ddaa737ba79eb4467a61eba5baf30) when adding it to the main app.)